service/flow_helpers/account.go

```diff
@@ -34,10 +36,15 @@ func GetAccount(address flow.Address, privateKeyInHex string, keyIndexes []int)
 		return existing
 	}
 
+	// Pick a random index to start from
+	rand.Seed(time.Now().UnixNano())
+	randomIndex := rand.Intn(len(keyIndexes))
+
 	new := &Account{
-		Address:         address,
-		PrivateKeyInHex: privateKeyInHex,
-		KeyIndexes:      keyIndexes,
+		Address:           address,
+		PrivateKeyInHex:   privateKeyInHex,
+		KeyIndexes:        keyIndexes,
+		nextKeyIndexIndex: randomIndex,
```

Comment:
This _looks_ like a typo but probably isn't 👀

Reply:
Yea... it is not a typo :)